prpr/filters.py

```diff
@@ -80,7 +80,8 @@ def filter_homeworks(
     if problems:
         result = [h for h in result if h.problem in problems]
     if student:
-        result = [h for h in result if student in h.student]
+        search_string = student.lower()
+        result = [h for h in result if search_string in h.student.lower()]
```

Comment:
```suggestion
        result = [h for h in result if student.lower() in h.student.lower()]
```

Reply:
Let's inline the search string.

Reply:
Yuri, thanks for the suggestion! Could you please update the change log accordingly?

Reply:
Done.